src/Property.cpp

```diff
@@ -31,9 +31,11 @@ void Property<T>::set_value(T val) {
 
 template <typename T>
 const T &Property<T>::get() {
+  auto tmp_attr =
```

Comment:
It seems like an unnecessary variable!

Reply:
done 5c904ed3bd1e4de5104225562791891596d6803c